src/v2/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -202,6 +202,7 @@ export const PaymentRoute: FC<Props> = props => {
     })
   }
 
+  const saveAccount = getClientParam("saveAccount")
```

Comment:
#non-blocking: noticed inconsistency of the casing of the query params, i.e. `saveAccount` vs. `setup_intent` below. Not sure what's the recommended casing but would be good to be consistent I think.